lib/verdeps/find_package_import_comment_test.go

```diff
@@ -0,0 +1,80 @@
+package verdeps
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFindPackageImportComment(t *testing.T) {
+	Convey("Given file data and a start index", t, func() {
+		Convey("When there is a package import comment, its indices should be returned", func() {
+			var (
+				fileData = []byte(`
+/**
```

Comment:
Can we move this string outside the test (a constant) because it makes this kinda gross to read